deploy/sample/percona.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: openebs-zfspv
 allowVolumeExpansion: true
 parameters:
-  blocksize: "4k"
+  recordsize: "4k"
```

Comment:
should we support both depending on zfs or zvol? to match the zfs naming convention?

Reply:
I am missing C union here :). I feel recordsize for block devices will mean volblock size only. Just keeping both the field for both of them does not look good.

Reply:
it might get confusing to have recordsize for zvol.  recordsize and volblocksize are better. Each should have a default value and be optional.

Reply:
hmmm, so while doing zfs property change, user will see both volblocksize and recordsize.

Reply:
added volblocksize and recordsize both. They will make sese for zvol and dataset respectively.